### PR TITLE
Add account type details and bugfix

### PR DESCRIPTION
**Pull request by rafael-azevedo**
This PR fixes the return of the getSTSCredentials function and adds the spec and state details of the Account_type.

### CODE REVIEW

**Comment by jharrington22** [comment]
@rafael-azevedo can you add proper descriptions to these PR's please

File: pkg/apis/aws/v1alpha1/account_types.go [hunk 1]
```diff
@@ -13,6 +13,9 @@ type AccountSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+	AwsAccountID  string `json:"awsaccountid"`
+	IAMUserSecret string `json:"iamusersecret"`
+	Claim         string `json:"claim"`
```
Comment (jharrington22) [inline pkg/apis/aws/v1alpha1/account_types.go]: Can we make rename this to `ClaimLink` please

Comment (rafael-azevedo) [reply pkg/apis/aws/v1alpha1/account_types.go]: updated

File: pkg/apis/aws/v1alpha1/account_types.go [hunk 2]
```diff
@@ -13,6 +13,9 @@ type AccountSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+	AwsAccountID  string `json:"awsaccountid"`
+	IAMUserSecret string `json:"iamusersecret"`
+	ClaimLink     string `json:"claim"`
```
Comment (jharrington22) [inline pkg/apis/aws/v1alpha1/account_types.go]: `json:"claimlink"`?

Comment (rafael-azevedo) [reply pkg/apis/aws/v1alpha1/account_types.go]: fixed

**Pull request merged**